apps/src/sites/studio/pages/incubator/index.js

```diff
@@ -0,0 +1,15 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+import {getStore} from '@cdo/apps/code-studio/redux';
+import Incubator from '@cdo/apps/templates/studioHomepages/Incubator';
+
+$(document).ready(function() {
+  ReactDOM.render(
+    <Provider store={getStore()}>
```

Comment:
are we using redux at all in the `Incubator` component?

Reply:
Great question.  The `HeaderBanner` component apparently [needs](https://github.com/code-dot-org/code-dot-org/blob/f8c00060368b0c9f9cf0132e0a7cfce8f58949b9/apps/src/templates/HeaderBanner.jsx) it.